docs/notable-contributors.md

```diff
@@ -16,3 +16,4 @@ If there is someone missing that you think should be on this list, send a PR on
 - EmreTech (Gentoo) [https://github.com/EmreTech](https://github.com/EmreTech)
 - T2minator (Tails) [https://github.com/T2minator](https://github.com/T2minator)
 - NoaHimesaka1873 (Arch, Manjaro and EndeavourOS) [https://github.com/NoaHimesaka1873](https://github.com/NoaHimesaka1873)
+- The Asahi Linux Contributors (Helped in providing the Wi-Fi patches for T2 Macs) [https://asahilinux.org/](https://asahilinux.org/)
```

Comment:
Would "writing" the wifi patches be better than "providing"? You could also probably just say "The Asahi Linux Contributors (Wi-Fi)" if you want